views/aggregate-block-metadata.json

```diff
@@ -7212,13 +7212,13 @@
             "block_schema_references": {}
           },
           "capabilities": [],
-          "version": "0.2.0"
+          "version": "0.2.1"
         }
       },
       "github-repository": {
         "name": "GitHub Repository",
         "slug": "github-repository",
-        "logo_url": "https://cdn.sanity.io/images/3ugk85nk/production/41971cfecfea5f79ff334164f06ecb34d1038dd4-250x250.png",
```

Comment:
dang @desertaxle - i assume this is a problem again? how can we prevent this? i added this [issue](https://github.com/PrefectHQ/prefect-collection-registry/issues/317) which im happy to take - im just not sure at the moment why this keeps happening / how you've previously fixed this

Reply:
The URLs in the source code still need to be updated. Maybe we could use the collection update tooling to mass update all the URLs?

Reply:
gotcha - yeah I'm sure we could. i can take a look at that